cfme/tests/services/test_provision_stack.py

```diff
@@ -149,19 +164,18 @@ def _cleanup(appliance=None, provision_request=None, service=None):
         myservice.delete()
 
 
-def test_provision_stack(appliance, stack, service_catalogs, request):
+def test_provision_stack(order_stack):
     """Tests stack provisioning
 
     Metadata:
         test_flag: provision
     """
-    provision_request = service_catalogs.order()
-    provision_request.wait_for_request(method='ui')
-    request.addfinalizer(lambda: _cleanup(appliance, provision_request))
+    provision_request, stack = order_stack
     assert provision_request.is_succeeded()
-    stack.wait_for_exists()
 
 
+@pytest.mark.uncollectif(lambda: BZ(1575935, forced_streams=['5.8', '5.9']).blocks,
```

Comment:
Required: Please use @pytest.mark.meta(blockers=[BZ(1575935, forced_streams=['5.8', '5.9'])]) as this will be marked in run as blocked be bz